internal/kptfile/pkgfile.go

```diff
@@ -121,18 +121,41 @@ func (updatedKf *KptFile) MergeOpenAPI(localKf, originalKf KptFile) error {
 	}
 	oriDef := original.Field("definitions")
 
-	err = localDef.Value.VisitFields(func(node *yaml.MapNode) error {
-		key := node.Key.YNode().Value
-		if shouldSkipCopy(updatedDef, localDef, oriDef, key) {
-			return nil
+	localKeys, _ := localDef.Value.Fields()
```

Comment:
The file is not lint checked. Run gofmt -s -w internal/kptfile/pkgfile.go to format the file. Wondering how it passed the checks.

Reply:
Done.